Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -458,4 +458,13 @@ private List<TraitorRole> GetOtherTraitorsAliveAndConnected(Mind.Mind ourMind, T
             .Where(t => _mobStateSystem.IsAlive((EntityUid) t.Mind.OwnedEntity!)) // dead
             .Where(t => t.Mind.CurrentEntity == t.Mind.OwnedEntity).ToList(); // not in original body
     }
+
+    /// <summary>
+    /// Manually adds a role to the list of traitors.
+    /// Use if you have custom logic for being a traitor, like ninja.
+    /// </summary>
+    public void AddToTraitors(TraitorRuleComponent rule, TraitorRole role)
+    {
+        rule.Traitors.Add(role);
+    }
```

Comment:
ninjas should have special end round text anyways (something like "syndicate contrator" instead of "traitor") so i dont see much use in this

Reply:
then id have to copy paste the entire traitor round end code and tweak a single line

Reply:
My brother how dare you make me see the atrocity that is `TraitorRuleSystem.OnRoundEndText()`

Reply:
AAAAAAAAAAAAAAAAAAHHHHHH OBJECTIVES ARE TIED TO TRAITORS. Why did i wipe this out of my memory

Reply:
Okay i fucking despise this with every bit of my soul, you can keep this accursed thing but please add a TODO as a xmldoc remark to this method to kill once objectives get separated from traitors

Reply:
Okay I despise this with every bit of my soul, you can keep this method but please add a TODO as a xmldoc remark to it to kill it once objectives round end text gets separated from traitors